com.viewizard.AstroMenace.json

```diff
@@ -0,0 +1,66 @@
+{
+  "app-id": "com.viewizard.AstroMenace",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "astromenace",
+  "rename-appdata-file": "astromenace.appdata.xml",
+  "rename-desktop-file": "astromenace.desktop",
+  "rename-icon": "astromenace",
+  "finish-args": [
+    "--socket=x11",
```

Comment:
You also need --share=ipc here for X11 to work properly

Reply:
fixed